config/brakeman.ignore

```diff
@@ -1,83 +1,23 @@
-      "warning_code": 18,
-      "fingerprint": "013f4e7ea9d88292d04e9cf18eb4ca634e75969ced93fd8752277b6d91428d99",
-      "check_name": "Redirect",
-      "message": "Possible unprotected redirect",
-      "file": "app/controllers/planning_applications_controller.rb",
-      "line": 208,
-      "link": "https://brakemanscanner.org/docs/warning_types/redirect/",
-      "code": "redirect_to(current_local_authority.planning_applications.find(params[:id]), :notice => \"Decision Notice sent to applicant\")",
-      "render_path": null,
-      "location": {
-        "type": "method",
-        "class": "PlanningApplicationsController",
-        "method": "determine"
-      },
-      "user_input": "params[:id]",
-      "confidence": "Weak",
-      "note": ""
-    },
-    {
-      "warning_type": "Redirect",
-      "warning_code": 18,
-      "fingerprint": "12c68226952c9a92bc6b0e207c6069435291ca4d92a73f8927a6cbb673dc9f7d",
-      "check_name": "Redirect",
-      "message": "Possible unprotected redirect",
-      "file": "app/controllers/planning_applications_controller.rb",
-      "line": 225,
-      "link": "https://brakemanscanner.org/docs/warning_types/redirect/",
-      "code": "redirect_to(current_local_authority.planning_applications.find(params[:id]), :notice => \"Site boundary has been updated\")",
-      "render_path": null,
-      "location": {
-        "type": "method",
-        "class": "PlanningApplicationsController",
-        "method": "update_sitemap"
-      },
-      "user_input": "params[:id]",
-      "confidence": "Weak",
-      "note": ""
-    },
-    {
-      "warning_type": "Redirect",
-      "warning_code": 18,
-      "fingerprint": "1ce6e33c46d250abc9d2297cdac112a81b2623e447d13e2aa0d90a0492c2be63",
-      "check_name": "Redirect",
-      "message": "Possible unprotected redirect",
-      "file": "app/controllers/planning_applications_controller.rb",
-      "line": 137,
-      "link": "https://brakemanscanner.org/docs/warning_types/redirect/",
-      "code": "redirect_to(current_local_authority.planning_applications.find(params[:id]), :notice => \"Application has been invalidated and email has been sent\")",
-      "render_path": null,
-      "location": {
-        "type": "method",
-        "class": "PlanningApplicationsController",
-        "method": "invalidate"
-      },
-      "user_input": "params[:id]",
-      "confidence": "Weak",
-      "note": ""
-    },
-    {
-      "warning_type": "Redirect",
-      "warning_code": 18,
-      "fingerprint": "3c8044aad0dfc6077d787528ad6b41c1869e156860948fd78d088ae46fd11e6f",
-      "check_name": "Redirect",
-      "message": "Possible unprotected redirect",
-      "file": "app/controllers/planning_applications_controller.rb",
-      "line": 156,
-      "link": "https://brakemanscanner.org/docs/warning_types/redirect/",
-      "code": "redirect_to(current_local_authority.planning_applications.find(params[:id]))",
+      "warning_type": "Mass Assignment",
```

Comment:
This is raised by `params.require(:user).permit(:role)`. Although it's not breaking the basic rule, since we're explicitly whitelisting the params, apparently 'Brakeman will warn on potentially dangerous attributes that are whitelisted.'

See https://brakemanscanner.org/docs/warning_types/mass_assignment/